### PR TITLE
Feaature/favicon support

### PR DESCRIPTION
**Pull request by danial24**
Added FavIcon Plugin in web client configs

The Fav Icon path is ./assets/web/icon.png
User will replace this icon to override the FavIcon

### CODE REVIEW

File: packages/web/src/webpack/middlewares/FavIcon.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const FavIcon: WebpackBuilderMiddleware =
+	() =>
+	(config: WebpackConfig): WebpackConfig => {
+
+		const newConfig: any = merge(config, {
+
+			plugins: removeNil([
+
+			// favIconPlugin
+                    new FaviconsWebpackPlugin({logo: './assets/web/icon.png'})
```
Comment (artalat) [inline packages/web/src/webpack/middlewares/FavIcon.ts]: rather than hardcoding this path can you make it configurable through `client.config.ts` file?

Comment (danial24) [reply packages/web/src/webpack/middlewares/FavIcon.ts]: ok will do that

**Pull request merged**